layout/_third-party/analytics/analytics-with-widget.swig

```diff
@@ -0,0 +1,4 @@
+{% include 'busuanzi-counter.swig' %}
+{% include 'busuanzi-counter.swig' %}
+{% include 'tencent-mta.swig' %}
```

Comment:
Are u tested `tencent-mta.swig`? I'm not sure, what is visible widget. Please, test it to be sure and also need to test `tencent.swig`.